### PR TITLE
Publish C++ stubs.

### PR DESCRIPTION
**Pull request by anuraaga**
I will publish to `stable` and then update README after @reiinakano gets a chance to try these out.

### CODE REVIEW

File: stubs/cpp/package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from conans import ConanFile, CMake
+
+
+class StellarstationapiConan(ConanFile):
```
Comment (hoshir) [inline stubs/cpp/package/conanfile.py]: Cases of class name looks not usual. Is this intentional?

File: stubs/cpp/package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    def package(self):
+        self.copy("*.h", dst="include", src="src")
+        self.copy("*.lib", dst="lib", keep_path=False)
+        self.copy("*.dll", dst="bin", keep_path=False)
+        self.copy("*.dylib*", dst="lib", keep_path=False)
+        self.copy("*.so", dst="lib", keep_path=False)
+        self.copy("*.a", dst="lib", keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = ["hello"]
```
Comment (hoshir) [inline stubs/cpp/package/conanfile.py]: Is this correct?

File: stubs/cpp/package/conanfile.py (hunk 1 above)
Comment (anuraaga) [reply stubs/cpp/package/conanfile.py]: It was automatically generated by conan, but seems like we can rename it.

File: stubs/cpp/package/conanfile.py (hunk 2 above)
Comment (anuraaga) [reply stubs/cpp/package/conanfile.py]: Thanks!

**Pull request merged**